### ISSUE TITLE
cannot unmarshal string into Go struct field AccessDeviceInfo.auth_context.access_device.is_encryption_enabled

### ISSUE THREAD

**Issue by hazcod** [duosecurity/duo_universal_golang#4]
Hi  @AaronAtDuo & @jeffreyparker ! When testing this code I am getting following error:

```
"json: cannot unmarshal string into Go struct field AccessDeviceInfo.auth_context.access_device.is_encryption_enabled of type bool" username=foo@company.com
```

When calling:
```go
duoAuth, err := duo.ExchangeAuthorizationCodeFor2faResult(urlDuoCode, token.Subject)
```

**Comment by wneessen** [comment]
I am running into the same error. I had a look at the HTTP response from the Duo API and it looks like the Base64-encoded JSON inside the `id_token` part of the response fails to decode the JSON for "remembered device" authentication in https://github.com/duosecurity/duo_universal_golang/blob/main/duouniversal/duo_jwt.go#L60

Looks some fields like i. e. `auth_context.access_device.is_encryption_enabled` is expecting a boolean value (see: https://github.com/duosecurity/duo_universal_golang/blob/main/duouniversal/client.go#L86), while the API response set the value to "unknown" (maybe b/c it doesn't have the information due to the "remembered login").

Not quite sure what's the best approach to handle this.

**Comment by wneessen** [comment]
As a quick and dirty workaround, I change the type for https://github.com/duosecurity/duo_universal_golang/blob/main/duouniversal/client.go#L86 thru https://github.com/duosecurity/duo_universal_golang/blob/main/duouniversal/client.go#L88 from "bool" to "string" in my local copy of `client.go`. 

This works so far, as it can now unmarshal the "unknown" but might be not the desired result, as now a simple boolean comparsion against i. e. `authToken.AuthContext.AccessDevice.IsEncryptionEnabled` won't work anymore but needs to be changed to a string comparsion.

Maybe the DEVs can come up with a nicer solution.

**Comment by hazcod** [comment]
Thanks @wneessen , can you open up a PR to this repo?

**Comment by wneessen** [comment]
https://github.com/duosecurity/duo_universal_golang/pull/5

**Comment by AaronAtDuo** [comment]
Should be fixed by 5c3fe79f3e78aaaa5667387f7196d1473dd54deb.  Thanks for the help with this!
